libpod/define/container_inspect.go

```diff
@@ -189,22 +189,28 @@ type InspectMount struct {
 // Docker, but here we see more fields that are unused (nonsensical in the
 // context of Libpod).
 type InspectContainerState struct {
-	OciVersion   string             `json:"OciVersion"`
-	Status       string             `json:"Status"`
-	Running      bool               `json:"Running"`
-	Paused       bool               `json:"Paused"`
-	Restarting   bool               `json:"Restarting"` // TODO
-	OOMKilled    bool               `json:"OOMKilled"`
-	Dead         bool               `json:"Dead"`
-	Pid          int                `json:"Pid"`
-	ConmonPid    int                `json:"ConmonPid,omitempty"`
-	ExitCode     int32              `json:"ExitCode"`
-	Error        string             `json:"Error"` // TODO
-	StartedAt    time.Time          `json:"StartedAt"`
-	FinishedAt   time.Time          `json:"FinishedAt"`
-	Health       HealthCheckResults `json:"Health,omitempty"`
-	Checkpointed bool               `json:"Checkpointed,omitempty"`
-	CgroupPath   string             `json:"CgroupPath,omitempty"`
+	OciVersion     string             `json:"OciVersion"`
+	Status         string             `json:"Status"`
+	Running        bool               `json:"Running"`
+	Paused         bool               `json:"Paused"`
+	Restarting     bool               `json:"Restarting"` // TODO
+	OOMKilled      bool               `json:"OOMKilled"`
+	Dead           bool               `json:"Dead"`
+	Pid            int                `json:"Pid"`
+	ConmonPid      int                `json:"ConmonPid,omitempty"`
+	ExitCode       int32              `json:"ExitCode"`
+	Error          string             `json:"Error"` // TODO
+	StartedAt      time.Time          `json:"StartedAt"`
+	FinishedAt     time.Time          `json:"FinishedAt"`
+	Health         HealthCheckResults `json:"Health,omitempty"`
+	Checkpointed   bool               `json:"Checkpointed,omitempty"`
+	CgroupPath     string             `json:"CgroupPath,omitempty"`
+	CheckpointedAt time.Time          `       json:"CheckpointedAt,omitempty"`
```

Comment:
This looks like it could cause an issue.

Reply:
I removed the unnecessary whitespace.